docs/send-data/collector-faq.md

```diff
@@ -179,9 +179,12 @@ Is it possible to delete data already collected into Sumo Logic? I've ingested
 
 #### Answer
 
-It is not possible for users to delete specific message data already ingested into Sumo Logic. All data sent to Sumo Logic is indexed together and stored in a Write Once Read Many (WORM) storage, which you cannot modify. You may request data be deleted from your account. If you have Views and Partitions set up, Sumo Logic can delete data from that specific View or Partition. Otherwise, the deletion will cover ALL data delivered into the account within a specified time range. Any request for data deletion should be sent to [Sumo Logic Support](mailto:support@sumologic.com) through your account administrator.
-
-An alternative to requesting Sumo Logic to delete your message data is to create Role filters that will hide the unwanted data so it is not searchable within the account. To do this, you can apply a role filter query string matching this data to the user roles for whom the unwanted data should not be visible.
+It is not possible for users to delete specific message data already ingested into Sumo Logic. But Sumo Logic provides two mechanisms for data deletion, with which you can hide the data instead of deleting it when users search for the same.
```

Comment:
Sumo Logic does not support deletion of specific log messages. You can either hide specific log messages or choose to delete the entire data ingested in a given time range & index.